app/models/provider.rb

```diff
@@ -108,6 +108,9 @@ def current_accredited_courses
 
   validates :provider_name, length: { maximum: 100 }, on: :update, if: RecruitmentCycle.current_recruitment_cycle
 
+  validates :urn, length: { in: 5..6 }, unless: -> { urn.blank? }
+  validates :urn, presence: true, if: -> { provider_type == "lead_school" }
```

Comment:
ummm not too sure....

only apply validation if `lead_school` and for it to be 5 or 6 in length and allow for nil...

Reply:
Done 👍